.github/workflows/docstyle.yaml

```diff
@@ -0,0 +1,13 @@
+name: Run docstyle
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  call-workflow:
+    uses: lsst/rubin_workflows/.github/workflows/docstyle.yaml@main
+    with:
+      args: "python/lsst/daf/butler/core/"
```

Comment:
I was confused as to how it was that your pydocstyle run was passing without there being any pydocstyle configuration in your pyproject.toml and it's because this args path is not correct (and also I see there is no code...).

https://github.com/lsst/daf_butler/blob/main/pyproject.toml#L126-L134

Reply:
Added this block to `pyproject.toml`:
```
[tool.pydocstyle]
convention = "numpy"
# Our coding style does not require docstrings for magic methods (D105)
# Our docstyle documents __init__ at the class level (D107)
# We allow methods to inherit docstrings and this is not compatible with D102.
# Docstring at the very first line is not required
# D200, D205 and D400 all complain if the first sentence of the docstring does
# not fit on one line.
add-ignore = ["D107", "D105", "D102", "D100", "D200", "D205", "D400"]
```

What should the args path be in this case?

Reply:
`args` changed to: 
```
args: "python/lsst/source/injection"
```